rasterio/__init__.py

```diff
@@ -50,135 +51,13 @@ def open(
         dtype=None,
         nodata=None,
         **kwargs):
-    """Open file at ``path`` in ``mode`` "r" (read), "r+" (read/write),
-    or "w" (write) and return a ``Reader`` or ``Updater`` object.
+    """See Env.open()"""
+    env = Env()
```

Comment:
@sgillies On [second look](https://github.com/mapbox/rasterio/pull/665/files#r60008227) I think this is correct and `Env.open()` shouldn't be receiving an optional environment since it is already attached to one.